pom.xml

```diff
@@ -23,6 +23,7 @@
     </scm>
 
     <modules>
+		<module>reader</module>
```

Comment:
I fixed the tabs issue. However, if I enable the checkstyle on this project, the build fails due to the fact that I'm using lambda expressions.